### ISSUE TITLE
explain how the context attention is "optimized"

### ISSUE THREAD

**Issue by iranroman** [dvlab-research/LLaMA-VID#5]
Hello,

Thanks a lot for your great work. This model truly is a leap forward. And thanks a lot as well for sharing the code and trained models. I have been having a lot of fun with these. 

I have a question about the paper. In section 3.3 "Modality alignment", you say that you freeze the "pre-trained modules like the visual encoder and text decoder". You also say that you "primarily optimize the context attention and projectors in Figure 2". However, what is there to optimize about the context attention? I consists of multiplication and standard normalization and averaging operations on the outputs of the frozen visual and text decoders. Are there other parameters involved in the context attention that I may be missing? 

Cheers!
Iran

**Comment by yanwei-li** [comment]
Hi Iran,

Sorry for the confusion. It's a typo. In the modality alignment, we mainly optimize the projectors, including the key and value projector for context attention like this (https://github.com/dvlab-research/LLaMA-VID/blob/c685cfb9dab9c66e77e221af0a8bf498f4e467af/llamavid/model/llamavid_arch.py#L106).
There are no other parameters in it.

Regards,
Yanwei

**Comment by yanwei-li** [comment]
Close it now, please reopen it if you have further questions.
